lib/matplotlib/collections.py

```diff
@@ -415,7 +415,7 @@ def set_hatch(self, hatch):
         can only be specified for the collection as a whole, not separately
         for each member.
 
-        ACCEPTS: [ '/' | '\\\\' | '|' | '-' | '+' | 'x' | 'o' | 'O' | '.' | '*' ]
+        ACCEPTS: [ '/' | '\\' | '|' | '-' | '+' | 'x' | 'o' | 'O' | '.' | '*' ]
```

Comment:
It appears that this line is under 79 characters and this file now passes PEP8, so it needs to be removed from `lib/matplotlib/tests/test_coding_standards.py` because the build is failing.